cmd/search/static-services-entries.json

```diff
@@ -1004,8 +1004,8 @@
       {
         "custom": true,
         "id": "roadmap.lifecycle",
-        "href": "/insights/roadmap/lifecycle",
-        "title": "Lifecycle",
+        "href": "/insights/planning/lifecycle",
```

Comment:
There is probably another thing happening, as we are moving from the `fed`config and doing feo migration, do we want to have the #853 #854 and #856 for chrome service, or it should be plased elsewhere?

Reply:
@Hyperkid123  No need for merging this PR, the migration already happened, and the result is here https://github.com/RedHatInsights/digital-roadmap-frontend/blob/main/deploy/frontend.yaml (if you want to take a look). I will create PR for reverting #853 so it's not duplicit.

Reply:
Kudos on using the CR!